### PR TITLE
Anim events user manual page

### PR DESCRIPTION
**Pull request by ellthompson**
image

### CODE REVIEW

File: content/en/user-manual/animation/anim-events.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+After creating an event for an animation asset, the event will be fired whenever that asset is played back by an anim component. You can therefore attach listeners to the anim component to handle the event. The following example shows how to attach event listeners to the anim component:
+```javascript
+var AnimationEvents = pc.createScript('animationEvents');
+
+// initialize code called once per entity
+AnimationEvents.prototype.initialize = function() {
+    this.entity.anim.on('plant_foot', function (event) {
+        // get the foot bone from the event's string property, either right_foot or left_foot
+        const footBone = this.entity.findByName(event.string);
```
Comment (willeastcott) [inline content/en/user-manual/animation/anim-events.md:51]: Won't `this` be wrong unless you use an arrow function? Or pass `this` as the third parameter of `on`?

File: content/en/user-manual/animation/anim-events.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+<table class="table table-striped">
+    <tbody>
+        <tr>
+            <th>Variable</th>
+            <th>Description</th>
+        </tr>
+    <tr>
+        <td>time</td>
+        <td>Defines the specific time during the playback of the animation when the event should trigger. Given in normalised time based on the animation's duration.</td>
```
Comment (willeastcott) [inline content/en/user-manual/animation/anim-events.md:25]:
```suggestion
        <td>Defines the specific time during the playback of the animation when the event should trigger. Given in normalized time based on the animation's duration.</td>
```

**Review by willeastcott — APPROVED** [review]

**Comment by willeastcott** [comment]
Just for completeness, can you add a screenshot of your new docs to your PR description? Firefox lets you grab a screenshot of the whole page (on a right click, I think).

File: content/en/user-manual/animation/anim-events.md (hunk 1 above)
Comment (ellthompson) [reply content/en/user-manual/animation/anim-events.md:51]: Yep you're right, i'll update this.

**Pull request merged**